### PR TITLE
Add checks to constructors and add missing constructor members

### PR DESCRIPTION
**Pull request by jonahwilliams**

### CODE REVIEW

File: web_sdk/test/api_conform_test.dart [hunk 1]
```diff
@@ -39,10 +39,70 @@ void main() {
...
+          final FormalParameter uiParam =
+              uiConstructor.parameters.parameters[i];
+          final FormalParameter webParam =
+              webConstructor.parameters.parameters[i];
+          if (webParam.identifier.name != uiParam.identifier.name) {
+            failed = true;
+            print('Warning: lib/ui/ui.dart $className.$name parameter $i'
+                ' ${uiParam.identifier.name} has a different name in lib/stub_ui/ui.dart.');
+          }
+          if (uiParam.isPositional && !webParam.isPositional) {
```
Comment (jason-simmons) [inline web_sdk/test/api_conform_test.dart]: Should this be `uiParam.isPositional != webParam.isPositional`?

(same for `isNamed`)

Comment (jonahwilliams) [reply web_sdk/test/api_conform_test.dart]: Fixed

**Comment by GaryQian** [comment]
Docs for ` List<FontFeature> fontFeatures,` should be added.

**Comment by jonahwilliams** [comment]
Its okay to leave them out, because this code is essentially overwritten
when I sync

**Pull request merged**